starter-code/src/functions-and-arrays.js

```diff
@@ -1,100 +1,438 @@
 // Find the maximum
 
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a;
+  } else {
+    return b;
+  }
+}
+m;
+console.log(maxOfTwoNumbers(4, 2));
 // Finding Longest Word
+
 var words = [
-  'mystery',
-  'brother',
-  'aviator',
-  'crocodile',
-  'pearl',
-  'orchard',
-  'crackpot'
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
 ];
+var lgth = 0;
+
+for (var index = 0; index < words.length; index++) {
+  if (words[index].length > lgth) {
+    var lgth = words[index].length;
+    answer = words[index];
+  }
+}
+
+console.log(answer);
+
+function findLongestWord(words) {
+  words.forEach(function(wordsIndex) {
+    if (wordsIndex.length > lght) {
+      var lght = wordsIndex.length;
+      answer = wordsIndex;
+    }
+  });
+}
+console.log(answer);
 
```

Comment:
You should create your function first and and initialise the var inside the function. The variable is there to stock the longest word.
Then you define if the array is empty by an `if statement`. You could use a `forEach` but I think it's simpler with a `for loop`. 

Here is the way to do it: 
```
function findLongestWord(array) {
  var longest = wordsArr[0];
if (array.length === 0) {
  return undefined;
  }

for (var = i; i < array.length; i++) {
  if (array[i].length > longestWord.length){
    longestWord = array[i];
    return longestWord;
    }
  }
};
```